docs/reference/hostfacts/hostfacts.md

```diff
@@ -2,25 +2,24 @@
 title: Hostfacts
 menu:
   product_searchlight_4.0.0:
-    identifier: hostfacts-hostfacts
+    identifier: hostfacts
```

Comment:
Overwrites `docs/reference/hostfacts/_index.md` id.  ID should be unique.
I followed, Identifier: <folder/parent name>-<file-name> to avoid ID conflicts.